phenol-annotations/src/main/java/org/monarchinitiative/phenol/annotations/formats/hpo/HpoAnnotation.java

```diff
@@ -118,6 +129,11 @@ public boolean equals(Object that) {
         && modifiers.equals(otherHpoAnnotation.modifiers);
   }
 
+  public boolean isPCS() { return this.evidence.equals(EvidenceCode.PCS); }
```

Comment:
Comparing enums should use '==' rather than .equals()

Reply:
fixed, thanks!